File: ZMusic-Plugin/src/main/java/me/zhenxin/zmusic/command/CmdBukkit.java
package me.zhenxin.zmusic.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;
import org.bukkit.command.TabExecutor;

import java.util.List;

public class CmdBukkit implements TabExecutor {

    @Override
    public boolean onCommand(CommandSender sender, Command cmd, String label, String[] args) { //指令输出
        return Cmd.cmd(sender, args);
    }

    @Override
    public List<String> onTabComplete(CommandSender sender, Command cmd, String alias, String[] args) {
        return Cmd.tab(sender, args);
    }
}

File: src/main/kotlin/me/zhenxin/zmusic/command/CmdBukkit.kt
package me.zhenxin.zmusic.command

import me.zhenxin.zmusic.module.sender.SenderBukkit
import org.bukkit.command.Command
import org.bukkit.command.CommandSender
import org.bukkit.command.TabExecutor

class CmdBukkit : TabExecutor {
    override fun onTabComplete(
        sender: CommandSender,
        command: Command,
        alias: String,
        args: Array<String>
    ): MutableList<String> {
        return CmdEx.tab(SenderBukkit(sender), args)
    }

    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        return CmdEx.cmd(SenderBukkit(sender), args)
    }

}
